resources/profiles/Creality.json

```diff
@@ -182,10 +198,26 @@
             "name": "0.20mm Standard @Creality Ender6",
             "sub_path": "process/0.20mm Standard @Creality Ender6.json"
         },
+        {
+			"name": "0.20mm Standard @Creality 0.4 K1",
+			"sub_path": "process/0.20mm Standard @Creality 0.4 K1.json"
+		},
+		{
+			"name": "0.20mm Standard @Creality 0.4 K1Max",
```

Comment:
I prefer to name it as "xxx @Creality K1Max 0.4 nozzle"

Reply:
Thanks! Was renamed by yours pattern